src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/MockBlobClientExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿#nullable enable
+using System;
+using System.Threading;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Moq;
+using Moq.Language.Flow;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+
+public static class MockBlobClientExtensions
+{
+    public static IReturnsResult<BlobClient> SetupDownloadContentAsync(
+        this Mock<BlobClient> blobClient,
+        string content,
+        string? contentEncoding = null,
+        CancellationToken cancellationToken = default)
+    {
+        var downloadDetails = BlobsModelFactory.BlobDownloadDetails(
```

Comment:
Yikes!

Reply:
I saw at least 1 other issue raised about a similar factory method not having default methods making using them difficult, but until this one gets the same treatment I think we're stuck with this!